### ISSUE TITLE
site broken

### ISSUE THREAD

**Issue by metawops** [spech66/bootstrap-bp-hugo-theme#3]
After installing and selecting this theme my local site outputs this:
```
render of "taxonomyTerm" failed: execute of template failed: template: _default/terms.html:6:8: executing "_default/terms.html" at <partial "head.html" .>: error calling partial: "/Users/stefan/Hugo/modular/themes/bootstrap-bp-hugo-theme/layouts/partials/head.html:25:91": execute of template failed: template: partials/head.html:25:91: executing "partials/head.html" at <resources.Concat>: error calling Concat: slice []interface {} not supported in concat
render of "section" failed: execute of template failed: template: _default/list.html:6:8: executing "_default/list.html" at <partial "head.html" .>: error calling partial: "/Users/stefan/Hugo/modular/themes/bootstrap-bp-hugo-theme/layouts/partials/head.html:25:91": execute of template failed: template: partials/head.html:25:91: executing "partials/head.html" at <resources.Concat>: error calling Concat: slice []interface {} not supported in concat
render of "page" failed: execute of template failed: template: _default/single.html:6:8: executing "_default/single.html" at <partial "head.html" .>: error calling partial: "/Users/stefan/Hugo/modular/themes/bootstrap-bp-hugo-theme/layouts/partials/head.html:25:91": execute of template failed: template: partials/head.html:25:91: executing "partials/head.html" at <resources.Concat>: error calling Concat: slice []interface {} not supported in concat
render of "page" failed: execute of template failed: template: _default/single.html:6:8: executing "_default/single.html" at <partial "head.html" .>: error calling partial: "/Users/stefan/Hugo/modular/themes/bootstrap-bp-hugo-theme/layouts/partials/head.html:25:91": execute of template failed: template: partials/head.html:25:91: executing "partials/head.html" at <resources.Concat>: error calling Concat: slice []interface {} not supported in concat
failed to render pages: render of "home" failed: execute of template failed: template: index.html:6:8: executing "index.html" at <partial "head.html" .>: error calling partial: "/Users/stefan/Hugo/modular/themes/bootstrap-bp-hugo-theme/layouts/partials/head.html:25:91": execute of template failed: template: partials/head.html:25:91: executing "partials/head.html" at <resources.Concat>: error calling Concat: slice []interface {} not supported in concat
```

☹️

**Comment by spech66** [comment]
Do you have custom taxonomie categories (like categorie and tags)? If yes you need to add them to the translations:
https://github.com/spech66/bootstrap-bp-hugo-theme/blob/master/i18n/en.toml

Could you try this please. If this brakes the theme I have to add some checks for this in the next release, too.

**Comment by metawops** [comment]
Nope, no specialties. It's an out-of-the-box installation of Hugo, a new, blank site one (empty) post, one (empty) page. (I'm just starting with Hugo.)
However, I found the problem: my `config.toml` file contained a setting in the `[params]` section from a theme I tried earlier: `themeColor = "theme-base-09"`. When I comment this line out your theme works without errors.
So maybe there's a way to make your theme a bit more robust against those kind of settings?
